db/migrate/20170312180419_add_userid_to_answers.rb

```diff
@@ -0,0 +1,5 @@
+class AddUseridToAnswers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :answers, :user_id, :integer
+  end
+end
```

Comment:
You are correct. I has the relationship set up as a 1-to-n between users and answers. This is now changed to a n-to-n with `has_and_belongs_to_many`. This indeed required a join table migration.